featurebyte/api/view.py

```diff
@@ -93,6 +96,30 @@ def timestamp_column(self) -> Optional[str]:
             return None
         return self._parent.timestamp_column
 
+    @property
+    def cleaning_operations(self) -> Optional[List[CleaningOperation]]:
+        """
+        List of cleaning operations that were applied to this column during the view's creation.
+
+        Returns
+        -------
+        Optional[List[CleaningOperation]]
+            Returns None if the view column does not have parent. Otherwise, returns the list of cleaning operations.
+
+        Examples
+        --------
+        >>> view = catalog.get_view("GROCERYPRODUCT")
+        >>> view["ProductGroup"].cleaning_operations
+        []
+        """
+        if not self._parent:
+            return None
+
+        for column_cleaning_operations in self.parent.column_cleaning_operations:
```

Comment:
do we need to be consistent with `self.parent` here and `self._parent` (underscore) in the if check above?

Reply:
Will update here to make it more consistent.